test/NuGet.Core.Tests/NuGet.Protocol.Tests/Utility/PackageIdValidatorTests.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Moq;
+using NuGet.Common;
+using NuGet.Packaging;
+using Xunit;
+
+namespace NuGet.Protocol.Tests.Utility
+{
+    public class PackageIdValidatorTests
+    {
+        [Theory]
+        [InlineData("../contoso")]
+        [InlineData("contoso/../package")]
+        [InlineData("contoso/.?///?")]
+        public void Validate_InvalidId_Throws(string id)
```

Comment:
>Validate_InvalidId_Throws [](http://example.com/codeflow?start=20&length=25)

Can we also have a positive case in this UT class? i.e. the validation succeeds.